include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -50,6 +50,15 @@ POSSIBILITY OF SUCH DAMAGE.
 namespace libtorrent {
 namespace dht
 {
+	// This structure hold the relevant counters for the storage
+	struct TORRENT_EXPORT dht_storage_counters
+	{
+		boost::int64_t torrents;
```

Comment:
32 bit ints should be more than enough, yes, I followed the counter's type. I will change it.